diff --git a/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
index 1111111..2222222 100644
--- a/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
@@ -33,6 +33,7 @@ target_include_directories(
 if (REACTNATIVE_MERGED_SO)
   target_link_libraries(
           ${LIB_TARGET_NAME}
+          c++_shared
           fbjni
           jsi
           reactnative
@@ -40,6 +41,7 @@ if (REACTNATIVE_MERGED_SO)
 else()
   target_link_libraries(
           ${LIB_TARGET_NAME}
+          c++_shared
           fbjni
           folly_runtime
           glog
